asp.tf

```diff
@@ -7,12 +7,12 @@ locals {
 
 module "asp" {
   source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
-  location            = "${var.location}"
-  env                 = "${var.env}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
-  asp_capacity        = "${local.asp_capacity}"
-  asp_sku_size        = "${local.sku_size}"
-  asp_name            = "${var.product}"
-  ase_name            = "${local.ase_name}"
-  tag_list            = "${local.tags}"
+  location            = var.location
+  env                 = var.env
+  resource_group_name = azurerm_resource_group.rg.name
+  asp_capacity        = local.asp_capacity
+  asp_sku_size        = local.sku_size
+  asp_name            = var.product
+  ase_name            = local.ase_name
+  tag_list            = local.tags
```

Comment:
If app service plan is not used, this module can be removed. If this is removed, any reference to this module must be removed too.

Reply:
For the ASP plan we would like to keep that as it is, as we are not sure about usage and impact as part of these changes.